docs-src/basic_usage.rst

```diff
@@ -171,6 +171,7 @@ See `channels.info <https://api.slack.com/methods/channels.info>`_ for more info
 
 Joining a channel
 ------------------
+Channels are the social hub of most Slack teams. Here's how you hop into one:
 ::
```

Comment:
Sphinx renders them correctly, but I can convert them real quickly for consistency.